app/views/availabilities/index.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class="container-fluid">
+  <h1>Schedule Availability</h1>
+
+  <%= link_to 'Logout', destroy_user_session_path, method: :delete %>
+
+  <%= form_tag("/availabilities", method: "post") do %>
+    <table class="table table-sm">
+      <tr>
+        <th></th>
+        <% start_time = @start_time %>
+        <% end_time = @end_time %>
+        <% while start_time <= end_time do %>
+          <th colspan="2">
+            <span class="time-label">
+              <%= start_time.strftime("%H:%M") %>
+            </span>
+          </th>
+          <% start_time += 1.hour %>
+        <% end %>
+      </tr>
+      <% @availabilities.each do |availability_day| %>
+        <tr>
+          <td class="dayname"><%= availability_day[0].day[0..2] %></td>
```

Comment:
`[0..2]` is for what? :o

Reply:
for changing "Monday" to "Mon"

Reply:
Hmmmmm not sure how I feel haha. Then why also need `availability_day[0]`?

Maybe you can just separate it in a method in the `helpers`

Reply:
Changed to ABBR_DAYNAMES